lib/new_relic.ex

```diff
@@ -90,6 +90,22 @@ defmodule NewRelic do
   @spec stop_transaction() :: :ok
   defdelegate stop_transaction(), to: NewRelic.Transaction
 
+  @doc """
+  Start a new "Other" transaction and stop the transaction after execution of the given block.
+  The return value of the block is returned.
+  """
+  defmacro transaction(category, name, do: block) do
+    quote do
+      NewRelic.start_transaction(unquote(category), unquote(name))
+
+      res = unquote(block)
+
+      NewRelic.stop_transaction()
```

Comment:
```suggestion
      try do
        unquote(block)
      after
        NewRelic.stop_transaction()
      end
```

Should I consider crash?

Reply:
Crashes are already handled properly